integration-tests/drivers/TestDriver063/android/app/build.gradle

```diff
@@ -134,6 +134,9 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
         versionName "1.0"
+        
+        testBuildType System.getProperty('testBuildType', 'debug')  // This will later be used to control the test apk build type
```

Comment:
Nice

Reply:
I don't understand any of it, but it came from https://github.com/wix/Detox/blob/master/docs/Introduction.Android.md 😄